docs/en/stack/ml/anomaly-detection/ootb-ml-jobs.asciidoc

```diff
@@ -0,0 +1,757 @@
+[role="xpack"]
+[[ootb-ml-jobs]]
+== Modules with prebuilt {anomaly-detect} jobs
+
+This page lists the available modules with prebuilt {anomaly-detect} jobs. Every 
+module item in the list contains a name, a short description, information about 
```

Comment:
Should we also have a quick sentence/link that explains how you get these prebuilt jobs? E.g. Per http://stack-docs_843.docs-preview.app.elstc.co/guide/en/machine-learning/master/create-jobs.html "Kibana recognizes certain types of data in Elasticsearch and provides specialized anomaly detection job wizards for that context".